### PR TITLE
tiflash: add config item about tiflash io rate limit

### PR DESCRIPTION
**Pull request by lidezhu**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lidezhu** [comment]
#https://github.com/pingcap/docs/pull/4392

**Review by flowbehappy — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ tcp_port = tiflash tcp 服务端口
 http_port = tiflash http 服务端口
 mark_cache_size = 5368709120 # 数据块元信息的内存 cache 大小限制，通常不需要修改
 minmax_index_cache_size = 5368709120 # 数据块 min-max 索引的内存 cache 大小限制，通常不需要修改
+
+[storage]
+    bg_task_io_rate_limit = 0 # 控制后台任务每秒写入字节数，0 代表没有限制
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:43]:
```suggestion
    bg_task_io_rate_limit = 0 # 限制后台任务每秒写入的字节数，0 代表没有限制
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -38,6 +38,9 @@ tcp_port = tiflash tcp 服务端口
 http_port = tiflash http 服务端口
 mark_cache_size = 5368709120 # 数据块元信息的内存 cache 大小限制，通常不需要修改
 minmax_index_cache_size = 5368709120 # 数据块 min-max 索引的内存 cache 大小限制，通常不需要修改
+
+[storage]
+    bg_task_io_rate_limit = 0 # 限制后台任务每秒写入的字节数，0 代表没有限制
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:43]:
```suggestion
    bg_task_io_rate_limit = 0 # 控制后台任务每秒写入的字节数，0 代表没有限制
```

**Pull request merged**